docs/network-flow-visibility.md

```diff
@@ -0,0 +1,435 @@
+# Network Flow Visibility in Antrea
+
+## Table of Contents
+
+<!-- toc -->
+- [Overview](#overview)
+  - [Grafana Flow Collector](#grafana-flow-collector)
+    - [Purpose](#purpose)
+    - [About Grafana and ClickHouse](#about-grafana-and-clickhouse)
+    - [Deployment Steps](#deployment-steps)
+      - [Credentials Configuration](#credentials-configuration)
+      - [ClickHouse Configuration](#clickhouse-configuration)
+    - [Pre-built Dashboards](#pre-built-dashboards)
+      - [Flow Records Dashboard](#flow-records-dashboard)
+      - [Pod-to-Pod Flows Dashboard](#pod-to-pod-flows-dashboard)
+      - [Pod-to-External Flows Dashboard](#pod-to-external-flows-dashboard)
+      - [Pod-to-Service Flows Dashboard](#pod-to-service-flows-dashboard)
+      - [Node-to-Node Flows Dashboard](#node-to-node-flows-dashboard)
+      - [Network-Policy Flows Dashboard](#network-policy-flows-dashboard)
+    - [Dashboards Customization](#dashboards-customization)
+<!-- /toc -->
+
+## Overview
+
+[Antrea](design/architecture.md) is a Kubernetes network plugin that provides network
```

Comment:
As we move this stuff to Theia, this link could be broken. (as far as I can tell relative links only work in the current repo).

Also, it seems you've already used absolute links to antrea-io/antrea in the rest of the document